README.md

```diff
@@ -453,61 +344,55 @@ package manager via the `package-manager` configuration. An example `assets-comp
 
 ## CLI parameters
 
 ### Command: `compile-assets`
 
-| Param                        | Description                                                  |
+| Param | Description |
 | ---------------------------- | ------------------------------------------------------------ |
-| `--ignore-lock[=<packages>]` | Ignore lock for either all or specific packages. Multiple packages must be separated by a comma. |
-| `--env=<env>`                | Set the Asset Compiler "env" used to choose among multiple configuration variants. |
-| `--hash-seed=<seed>`         | Uses the provided seed when generating packages hash.<br />Must match the seed passed to `assets-hash --seed`. |
-| `--no-dev`                   | Simulate Assets Compiler auto-run after a Composer install/update with a `--no-dev` flag.<br />That causes the plugin to check for `$default-no-dev` as default "env" before fallback to `$default`. |
+| `--ignore-lock[=<packages>]` | Ignores locks for all or specific packages. Multiple packets must be comma-separated. |
+| `--env=<env>` | Specifies the asset compiler "env" which is used to select between multiple configuration variants. |
+| `--hash-seed=<seed>` | Uses the provided seed when generating the package hash.<br />Must match the seed passed to `assets-hash --seed`. |
+| `--no-dev` | Simulate auto-run of the Assets Compiler after a Composer installation/update with the `--no-dev` flag.<br />This causes the plugin to check for `$default-no-dev` as the default `env` before falling back to `$default`. |
 
 ### Command: `assets-hash`
 
-| Param           | Description                                                  |
+| Param | Description |
 | --------------- | ------------------------------------------------------------ |
-| `--seed=<seed>` | Causes generated packages' hash to change based on the seed.<br />Must match the seed passed to `compile-assets --hash-seed`. |
-| `--env`         | Set the Asset Compiler "env" which affects assets' hash.     |
-| `--no-dev`      | Simulate Assets Compiler auto-run after a Composer install/update with a `--no-dev` flag.<br />That affects assets' hash if package `script` configuration makes use of `$default-no-dev` as default "env". |
+| `--seed=<seed>` | Causes the hash of generated packages to change based on the seed.<br />Must match the seed passed to `compile-assets --hash-seed`. |
+| `--env` | Sets the asset compiler "env" which affects the hash of the assets. |
+| `--no-dev` | Simulate Assets Compiler auto-run after a Composer install/update with the `--no-dev` flag.<br />This affects the hash of the assets if the configuration of the `script` package uses `$default-no-dev` as the default "env". |
 
 ## Advanced topics
 
-### Packages definitions in root
+### Package definitions in the root directory
```

Comment:
Shoudl be _"in the root package"_, or _"in the root package configuration"_, because the point is that `packages` configuration property can only be used in root package configuration. (And no matter if root package uses `composer.json` or `assets-compiler.json`).